examples/decoding/plot_mixed_gambles_space_net.py

```diff
@@ -0,0 +1,45 @@
+"""
+SpaceNet on Jimura et al "mixed gambles" dataset.
+==================================================
+
+The segmenting power of SpaceNet is quite visible here.
```

Comment:
Otherwise, "it's like simulated annealing, without decreasing the temperature" -source: probably @GaelVaroquaux